app/views/admins/members/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1><%= link_to "New Member", "members/new", method: :get %></h1>
+
+<table border="1">
+  <% @members.each do |member| %>
+    <tr>
+      <td><%= member.last_name %></td>
+      <td><%= member.first_name %></td>
+      <td><%= member.email %></td>
+      <td><%= button_to "View", "members/#{member.id}", method: :get %></td>
+      <td><%= button_to "Edit", "members/#{member.id}/edit", method: :get %></td>
+      <td><%= button_to "Delete", "members/#{member.id}", method: :delete %></td>
```

Comment:
Instead of hardcoding URLs, we like to use the generated rails route helpers (eg. new_member_path, edit_member_path, etc., see https://github.com/calblueprint/roots-of-success/blob/master/app/views/application/_teacher_header.html.slim). You can run `rake routes` to see the possible URLs.